### ISSUE TITLE
Original Corkscrew ?

### ISSUE THREAD

**Issue by dregad** [bryanpkc/corkscrew#8]
FYI, Pat Padgett seems to have surfaced again, I stumbled upon this:
https://github.com/patpadgett/corkscrew

**Comment by h-vetinari** [comment]
Yeah, it was another GH-mirror that ended up being being [transferred](https://github.com/patpadgett/corkscrew/pull/3#issuecomment-424635055) back to him. In that discussion Pat commented:
> Thank you for finding me! It's been an amazing experience with corkscrew. The community has maintained this small bit of code since I have abandoned it; although hosting the original site for years until an unknown expiration allowed a domain squatter to snipe agroman.net.
> 
> To answer your question, there are many forks of corkscrew available even on github. I'd like to cherry pick and merge all of the great work that others have done since my initial scrapping together of this project. Even if you feel like your work has been insignificant, the most significant thing has been reaching back out to me -- and if you feel comfortable transferring this repo to me is a good start, I'd attribute your work (ie. digging up bones and identifying them as a dinosaur! haha!). Assuming the distros catchup with the relocation to github, your attribution should be good for professional and personal reasons.
> 
> TL;DR - Yes, please xfer it back. I'm going to add your name to contributors and make sure you are known within the project.

So the README here should be updated and ideally the changes since then raised as PRs.

OTOH, there hasn't been much activity https://github.com/patpadgett/corkscrew since the ownership transfer, maybe @patpadgett can give an update?

**Comment by bryanpkc** [comment]
If @patpadgett is going to take back maintainership, I'm happy to submit PRs to his repo and shut my fork down. I haven't had time myself to work on it either.
